demos/powersync-nextjs-demo/src/library/powersync/SupabaseConnector.ts

```diff
@@ -118,7 +118,7 @@ export class SupabaseConnector extends BaseObserver<SupabaseConnectorListener> i
   }
 
   async uploadData(database: AbstractPowerSyncDatabase): Promise<void> {
-    const transaction = await database.getNextCrudTransaction();
+    const transaction = await database.getCrudBatch(1000);
```

Comment:
Why this change?

Reply:
This was added as a test for the `getCrudBatch` feature. Previously the batch would only contain 1 entry for each invocation (if online). This was because we would trigger an upload on each `execute` method. The common SDK now debounces and queues uploads to some extent now.

Reply:
I see the difference between the two methods now. I reverted this change.